docs/src/osdk/reference/commands/profile.md

```diff
@@ -0,0 +1,75 @@
+# cargo osdk profile
+
+## Overview
+
+The profile command is used to collect stack traces when running the target
+kernel in QEMU. It attaches to the GDB server initiated with the run subcommand
+and collects the stack trace periodically. The collected information can be
+used to directly generate a flame graph, or be stored for later analysis using
+[the original flame graph tool](https://github.com/brendangregg/FlameGraph).
```

Comment:
Need to update the Dockerfile to install this tool and the OSDK doc to mention it as a prerequisite.

Do not bump the version, yet. PR #1396 also plans to bump the version.

Reply:
This is not a prerequisite! We are not forcing the user to use it. OSDK does produce a flame graph without it. It's done by a crate [inferno](https://crates.io/crates/inferno).

So I don't think that there's a need for installing the original tool in the Docker image.